mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -1744,6 +1895,44 @@ impl UtxoRpcClientOps for ElectrumClient {
         )
     }
 
+    fn list_unspent_group(&self, addresses: Vec<Address>, _decimals: u8) -> UtxoRpcFut<UnspentMap> {
+        let script_hashes = addresses
+            .iter()
+            .map(|addr| {
+                let script = output_script(addr, ScriptType::P2PKH);
```

Comment:
Maybe add a comment above the `list_unspent_group` fn that it only works for P2PKH and Segwit addresses, for now, no support for P2SH. I really don't like how the `output_script` fn works although I am the one who edited it :)
I might refactor `output_script` in the future, I think it should be implemented for the Address Struct.

Reply:
No comment is needed since there is an issue for this now https://github.com/KomodoPlatform/atomicDEX-API/issues/1270